src/components/cart/cart-view/cart-view.tsx

```diff
@@ -20,7 +20,7 @@ export const CartView = ({
     onItemQuantityChange,
     onItemRemove,
 }: CartViewProps) => {
-    if (cart.lineItems.length === 0) {
+    if (!cart || cart.lineItems.length === 0) {
```

Comment:
!cart?.lineItems?.length

Reply:
I like to be more explicit when it comes to checking length :)